packages/amplify-category-analytics/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "amplify-cli-core": "3.0.0",
+    "amplify-prompts": "2.3.0",
```

Comment:
why is new dependency part of this refactoring?

Reply:
In the refactor I replaced `context.print` with `printer` from `amplify-prompts` as the `context.print` is deprecated. It is a small change that I figured I can do as part of this refactor, let me know if you feel this shouldn't be here and I can keep the context.print instead

Reply:
sounds good.